.github/workflows/dev-checks.yml

```diff
@@ -0,0 +1,27 @@
+name: Development checks
+
+on:
+  pull_request:
+    branches: [ "main" ]
+
+jobs:
+  tests:
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        node-version: [16.x, 18.x]
```

Comment:
What about 17.x and 19.x? I'm not sure what the best practice is here, couldn't we also just use the version that will actually be run on the server? That's what we really care about... not sure what the best practice is here, but fewer job runs might be faster or will at least be more efficient.

Reply:
So the build is done with 1.16 currently (https://github.com/RedHatInsights/image-builder-frontend/blob/9be2ceaba7d402c8cac79fd9d55fac465e974551/build_deploy.sh#L12), and the releaser container has 1.18. I think it's good to test those two, but more can't hurt of course.

Reply:
Per @croissanne "Ok so the releaser container has 1.18, the frontend-build-container has 1.16 by default, but changeable by setting NODE_BUILD_VERSION, which is currently set to 1.16 for us. so testing both 1.16 and 1.18 will be fine"